assets/xml/objects/object_funen.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<Root>
     <File Name="object_funen" Segment="6">
-        <DList Name="object_funen_DL_0000D0" Offset="0xD0" />
+        <DList Name="object_funen_DL_0000D0" Offset="0xD0" /> <!-- Original name is "obj_smork_modelT" -->
```

Comment:
I wonder what the `T` in `modelT` means

Reply:
I'm almost certain it's `Transparent`. You can see the pattern in quite a few actors where they have `opaDList` and `xluDList` pointers, and they just render them in their separate buffers. Similarly, when you look at the original names for assets, you usually see `model` and `modelT`, with the `model` asset being assigned to the `opaDList` pointer and the `modelT` asset being assigned to the `xluDList` pointer.